tests/xharness/Logging/LogFile.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.IO;
+
+namespace Xharness.Logging {
+	public class LogFile : Log, ILogFile {
+		object lock_obj = new object ();
+		public string Path { get; private set; }
+		FileStream writer;
+		bool disposed;
+
+		public LogFile (ILogs logs, string description, string path, bool append = true)
+			: base (logs, description)
+		{
+			if (path == null)
+				throw new ArgumentNullException (nameof (path));
+			Path = path;
+			if (!append)
+				File.WriteAllText (path, string.Empty);
+		}
+
+		public override void Write (byte [] buffer, int offset, int count)
```

Comment:
This should check disposed first and, in case it is true ,throw OR out warning and return